### DISCUSSION TITLE
measured metrics

### DISCUSSION THREAD

**Discussion by aminbouhamed** [GoogleChrome/lighthouse#15262] [General]
hello is lighthouse can mesure those metrics (Time to First Byte (TTFB),Start Render Time,Fully Loaded Time,Time to Interactive ? ) if yes how can i configure it to do that please

**Comment by adamraine** [comment]
We have several debugging metrics in the Lighthouse JSON result:

```js
const {lhr} = await lighthouse('https://example.com');
lhr.audits['metrics'].details.items[0].interactive // Time to Interactive
lhr.audits['metrics'].details.items[0].timeToFirstByte // TTFB
```

The other metrics aren't computed in Lighthouse to my knowledge.

**Reply by aminbouhamed** [reply]
file:///c:/Users/bouha/Desktop/stage%20meg/script.js:8
runnerResult.audits['metrics'].details.items[0].interactive // Time to Interactive        
             ^

TypeError: Cannot read properties of undefined (reading 'audits')
    at file:///c:/Users/bouha/Desktop/stage%20meg/script.js:8:14
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
i tried the code but this error was raised

**Reply by adamraine** [reply]
Please try `runnerResult.lhr.audits['metrics'].details.items[0].interactive`
